app/subscriber/src/features/my-reports/edit/ReportEditActions.tsx

```diff
@@ -47,10 +47,34 @@ export const ReportEditActions = ({ disabled, updateForm }: IReportEditActionsPr
   const navigate = useNavigate();
   const { toggle: toggleRemove, isShowing: isShowingRemove } = useModal();
   const { toggle: toggleSend, isShowing: isShowingSend } = useModal();
+  const { toggle: toggleStartNewReport, isShowing: isShowingStartNewReport } = useModal();
+
   const [{ publishReportInstance }] = useReportInstances();
 
   const instance = values.instances.length ? values.instances[0] : undefined;
 
+  const [showStartNextReport, setShowStartNextReport] = React.useState(true);
+
+  React.useEffect(() => {
+    if (
+      showStartNextReport &&
+      !isShowingStartNewReport &&
+      !active?.startsWith(ReportMainMenuOption.Settings) &&
+      instance &&
+      instance.sentOn
+    ) {
+      setShowStartNextReport(false);
+      toggleStartNewReport();
```

Comment:
Force the user to make a choice when viewing a report that has already been sent.